bower.json

```diff
@@ -8,6 +8,6 @@
   "main": "quick-element.html",
   "license": "MIT",
   "dependencies": {
-    "polymer": "polymer/polymer#0.8-preview"
+    "polymer": "polymer/polymer#0.8"
```

Comment:
Actually, I think https://github.com/Polymer/polymer/releases/tag/v0.8.0-rc.2 might be newer ;)

It's a mad house!